### PR TITLE
app-emulation/crun: Bump to 0.14.1

### PR DESCRIPTION
**Pull request by robertgzr**
... and take over maintainership ;)

### CODE REVIEW

File: app-emulation/crun/crun-0.14.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+src_configure() {
+	econf \
+		--disable-criu \
+		$(use_enable bpf) \
+		$(use_enable caps) \
+		$(use_enable seccomp) \
+		$(use_enable systemd) \
+		$(usex static-libs '--enabled-shared  --enabled-static' '--enable-shared --disable-static' '' '')
+
+	cp -v ${DISTDIR}/libcrun.lds ${S}/
```
Comment (juippis) [inline app-emulation/crun/crun-0.14.1.ebuild:53]: This needs to happen in `src_prepare`, also this is an external command so it needs to be appended with `|| die "message what operation failed"`.

File: app-emulation/crun/crun-0.14.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+	if use doc ; then
+		emake generate-man
+	fi
+}
+
+src_install() {
+	emake "DESTDIR=${D}" install-exec
+	if use doc ; then
+		emake "DESTDIR=${D}" install-man
+	fi
```
Comment (juippis) [inline app-emulation/crun/crun-0.14.1.ebuild]: So does this install documentation, or manpages? Should the use flag be `doc` or `man`?

File: app-emulation/crun/crun-0.14.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+src_configure() {
+	econf \
+		--disable-criu \
+		$(use_enable bpf) \
+		$(use_enable caps) \
+		$(use_enable seccomp) \
+		$(use_enable systemd) \
+		$(usex static-libs '--enabled-shared  --enabled-static' '--enable-shared --disable-static' '' '')
```
Comment (juippis) [inline app-emulation/crun/crun-0.14.1.ebuild]:
```suggestion
local myeconfargs=(
	--disable-criu
	$(use_enable bpf)
	$(use_enable caps)
	$(use_enable seccomp)
	$(use_enable systemd)
	$(usex static-libs '--enabled-shared  --enabled-static' '--enable-shared --disable-static' '' '')
)

econf "${myeconfargs[@]}"
```

File: app-emulation/crun/crun-0.14.1.ebuild (hunk 2 above)
Comment (robertgzr) [reply app-emulation/crun/crun-0.14.1.ebuild]: It's the man1 page

**Comment by robertgzr** [comment]
thanks for the suggestions

**Comment by robertgzr** [comment]
do you know how to avoid this warning:
```
>>> Unpacking libcrun.lds to /var/tmp/portage/app-emulation/crun-0.14.1/work
unpack libcrun.lds: file format not recognized. Ignoring.
```

**Comment by juippis** [comment]
By defining `src_unpack` in the ebuild to only unpack that `.tar.gz`, not sure if it's worth it, but you may do it if you like it more.

**Comment by robertgzr** [comment]
@juippis is this how it works, you're not using the GH merge? should I just close this...

**Comment by juippis** [comment]
Woops, we usually close these automatically while merging but I forgot to add the tag. Sorry,
7409655e3a8d4991fe41abe1d20acd87b1b5b9f4

**Comment by robertgzr** [comment]
no worries :)

**Pull request closed without merging**